src/main/java/groupM/players/mcts/MCTSPlayer.java

```diff
@@ -0,0 +1,72 @@
+package groupM.players.mcts;
+
+import core.AbstractGameState;
+import core.AbstractPlayer;
+import core.actions.AbstractAction;
+import core.interfaces.IStateHeuristic;
+
+import java.util.List;
+import java.util.Random;
+
+
+/**
+ * This is a simple version of MCTS that may be useful for newcomers to TAG and MCTS-like algorithms
+ * It strips out some of the additional configuration of MCTSPlayer. It uses BasicTreeNode in place of
+ * SingleTreeNode.
+ */
+public class MCTSPlayer extends AbstractPlayer {
```

Comment:
Maybe give it a different name so the markers don't get confused and think we've copied MCTSPlayer when the instructions tell us not to?

Reply:
Will do this later on